src/redux/tests/accomReducer.test.js

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
Is there a reason this import was needed? I legit have no idea if it's required or not 😂

Reply:
Haha I don't think so. Some of the tests were failing in this branch, so I copy and pasted the test suite from another branch in this past and this snuck in with that. I'll remove it.